x-pack/test/alerting_api_integration/spaces_only/tests/alerting/event_log.ts

```diff
@@ -187,60 +194,83 @@ export default function eventLogTests({ getService }: FtrProviderContext) {
         outcome: 'failure',
         message: `alert execution failure: test.throw:${alertId}: 'abc'`,
         errorMessage: 'this alert is intended to fail',
+        status: 'error',
+        reason: 'execute',
       });
     });
   });
+}
+
+interface SavedObject {
+  type: string;
+  id: string;
+  rel?: string;
+}
+
+interface ValidateEventLogParams {
+  spaceId: string;
+  savedObjects: SavedObject[];
+  outcome?: string;
+  message: string;
+  errorMessage?: string;
+  status?: string;
+  actionGroupId?: string;
+  instanceId?: string;
+  reason?: string;
+}
+
+export function validateEvent(event: IValidatedEvent, params: ValidateEventLogParams): void {
+  const { spaceId, savedObjects, outcome, message, errorMessage } = params;
+  const { status, actionGroupId, instanceId, reason } = params;
 
-  interface SavedObject {
-    type: string;
-    id: string;
-    rel?: string;
+  if (status) {
+    expect(event?.kibana?.alerting?.status).to.be(status);
   }
 
-  interface ValidateEventLogParams {
-    spaceId: string;
-    savedObjects: SavedObject[];
-    outcome?: string;
-    message: string;
-    errorMessage?: string;
+  if (actionGroupId) {
+    expect(event?.kibana?.alerting?.action_group_id).to.be(actionGroupId);
   }
 
-  function validateEvent(event: IValidatedEvent, params: ValidateEventLogParams): void {
```

Comment:
for this diff section, best viewed with ignore whitespace, as I lifted some functions out of a function scope, to make them top-level, so they could be exported, so they could be reused by the security_and_spaces tests